### PR TITLE
Address Konrad's comments

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

**Comment by tpoterba** [comment]
yup I missed one!

**Comment by tpoterba** [comment]
OK, now there's no ":class`" anywhere in project files

File: python/hail/expr/expression.py [hunk 1]
```diff
@@ -3151,7 +3151,7 @@ def alt_alleles(self):
         --------
         .. doctest::
 
-            >>> eval_expr(variant.alt_allele())
+            >>> eval_expr(variant.alt_alleles())
```
Comment (catoverdrive) [inline python/hail/expr/expression.py]: does this have to be `variant.alt_alleles`?

Comment (tpoterba) [reply python/hail/expr/expression.py]: good catch! fixed.

**Pull request merged**